### PR TITLE
New File Added: Assignment1, Assignment3, Assignment4 and Assignment5

### PR DESCRIPTION
**Pull request by vishnukopnar**

### CODE REVIEW

**Review by Pranjali1999 — APPROVED** [review]

**Review by akash19972306 — APPROVED** [review]

**Review by Bhavika222 — APPROVED** [review]

**Review by Bhavika222 — APPROVED** [review]

**Review by Akash-ing — APPROVED** [review]

**Review by shivnanda6272 — APPROVED** [review]

File: Vishnu/HTML/Assignment2/Assignment2.html [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+        <hr>
+
+        <h2>Initial public offering</h2>
+        <p>Google's initial public offering (IPO) took place five years later, on 19th August 2004.<br>
+            At that time Larry Page, Sergey Brin, and Eric Schmidt agreed to work together at Google for 30 20<br>
+            years, until the year 2024.</p>
+        <h1>Thank You...</h1>
+    </body>
+</html>
```
Comment (shivnanda6272) [inline Vishnu/HTML/Assignment2/Assignment2.html]: add new line

File: Vishnu/HTML/Assignment3/ass2_contact.html [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+                us.Contact us.Contact us. Contact us.Contact us.Contact us.Contact
+                us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact
+                us.Contact us. Contact us.Contact us.Contact us.Contact us.Contact
+                us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact us.
+              </div>
+              <div style="text-align:center">
+                <h1>Thank you...</h1>
+            </div>
+    </body>
+</html>
```
Comment (shivnanda6272) [inline Vishnu/HTML/Assignment3/ass2_contact.html]: Add new end line after in all files

File: Vishnu_45/html/Assignment2/Assignment2.html [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+            there, was assigned to show him around.</p>
+        <p>By some accounts, they disagreed about nearly everything during that first meeting, but by the following
+             year they struck a .<br>
+            Working from their dorm rooms, they built a search engine that used links to determine the importance of
+            individual pages on the <b>World Wide Web</b>.</p>
+
+        <p>They called this search engine Backrub.</p>
+
+        <hr>
+
```
Comment (shivnanda6272) [inline Vishnu_45/html/Assignment2/Assignment2.html]: remove all extra spaces

**Review by shivnanda6272 — CHANGES_REQUESTED** [review]
Please make the changes

File: Vishnu/HTML/Assignment2/Assignment2.html (hunk 1 above)
Comment (vishnukopnar) [reply Vishnu/HTML/Assignment2/Assignment2.html]: Yes sir, I'll do that

File: Vishnu/HTML/Assignment3/ass2_contact.html (hunk 2 above)
Comment (vishnukopnar) [reply Vishnu/HTML/Assignment3/ass2_contact.html]: yes sir

File: Vishnu_45/html/Assignment2/Assignment2.html (hunk 3 above)
Comment (vishnukopnar) [reply Vishnu_45/html/Assignment2/Assignment2.html]: yes sir

File: Vishnu/HTML/Assignment2/Assignment2.html (hunk 1 above)
Comment (vishnukopnar) [reply Vishnu/HTML/Assignment2/Assignment2.html]: Sir, I have to push updated files on github, is it right?

**Review by shivnanda6272 — APPROVED** [review]

**Review by shivnanda6272 — APPROVED** [review]

**Comment by avinash0070** [comment]
Change the title
give proper name 
this title is not acceptable

File: Vishnu/HTML/Assignment1/Assign1.html [hunk 4]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
```
Comment (avinash0070) [inline Vishnu/HTML/Assignment1/Assign1.html]: need proper indentation

File: Vishnu/HTML/Assignment1/Assign1.html [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+<html>
+<head>
+<title>assignment_1</title>
+</head>
+<body>
+<h1>HTML</h1>
+
```
Comment (avinash0070) [inline Vishnu/HTML/Assignment1/Assign1.html]: remove extra line

File: Vishnu/HTML/Assignment1/Assign1.html [hunk 6]
```diff
@@ -0,0 +1,13 @@
...
+<h2>Introduction</h2>
+<p>HTML is a standard markup language for Web Pages.</p>
+
```
Comment (avinash0070) [inline Vishnu/HTML/Assignment1/Assign1.html]: remove extra line

File: Vishnu/HTML/Assignment1/Assign2.html [hunk 7]
```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
```
Comment (avinash0070) [inline Vishnu/HTML/Assignment1/Assign2.html]: Need proper indentation
remove all extra line

File: Vishnu/HTML/Assignment1/Assign3.html [hunk 8]
```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
```
Comment (avinash0070) [inline Vishnu/HTML/Assignment1/Assign3.html]: Need proper indentation
remove all extra line

File: Vishnu/HTML/Assignment1/Assign3.html [hunk 9]
```diff
@@ -0,0 +1,18 @@
...
+<h2>Introduction</h2>
+<p>HTML is a standard markup language for Web Pages.</p>
+<hr>
+<h2>HTML Editor</h2>
+<h3>Notepad or TextEdit.</h3>
+<p>Web pages can be created and modified by using professional HTML editors.<br>
+However, for learning HTML we recommend a simple text editor like Notepad (PC) or TextEdit (Mac).</p>
+</body>
+</html>
+
```
Comment (avinash0070) [inline Vishnu/HTML/Assignment1/Assign3.html]: remove this line

File: Vishnu/HTML/Assignment2/Assignment2.html [hunk 10]
```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
```
Comment (avinash0070) [inline Vishnu/HTML/Assignment2/Assignment2.html]: remove all extra lines

File: Vishnu/HTML/Assignment2/Assignment2.html (hunk 1 above)
Comment (avinash0070) [inline Vishnu/HTML/Assignment2/Assignment2.html]: Add new line

File: Vishnu/HTML/Assignment3/ass1.html [hunk 11]
```diff
@@ -0,0 +1,68 @@
...
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>list</title>
+        <style>
+            .disc-list {
+              list-style-type: disc;
+            }
```
Comment (avinash0070) [inline Vishnu/HTML/Assignment3/ass1.html]: add new line here

File: Vishnu/HTML/Assignment3/ass1.html [hunk 12]
```diff
@@ -0,0 +1,68 @@
...
+            .mean {
+              color: peru;
+            }
```
Comment (avinash0070) [inline Vishnu/HTML/Assignment3/ass1.html]: add new line here

File: Vishnu/HTML/Assignment3/ass1.html [hunk 13]
```diff
@@ -0,0 +1,68 @@
...
+                </ul>
+                <li>SYSTEM-2</li>
+                <ul class="disc-list">
+                    <li>Triveni</li>
+                    <li>Kalyani</li>
+                </ul>
+            </ol>
+        </ol>
+    </body>
+</html>
```
Comment (avinash0070) [inline Vishnu/HTML/Assignment3/ass1.html]: add new line

File: Vishnu/HTML/Assignment3/ass1.html [hunk 14]
```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
```
Comment (avinash0070) [inline Vishnu/HTML/Assignment3/ass1.html]: remove all extra lines

File: Vishnu/HTML/Assignment3/ass2_contact.html [hunk 15]
```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<!DOCTYPE html>
```
Comment (avinash0070) [inline Vishnu/HTML/Assignment3/ass2_contact.html]: why are you use <!DOCTYPE html> 2 times

File: Vishnu/HTML/Assignment3/ass2_contact.html (hunk 2 above)
Comment (avinash0070) [inline Vishnu/HTML/Assignment3/ass2_contact.html]: add new line

File: Vishnu/HTML/Assignment3/ass2_home.html [hunk 16]
```diff
@@ -0,0 +1,96 @@
+<!DOCTYPE html>
```
Comment (avinash0070) [inline Vishnu/HTML/Assignment3/ass2_home.html]: need proper indentation 
use new lines in css code

File: Vishnu/HTML/Assignment3/ass2_home.html [hunk 17]
```diff
@@ -0,0 +1,96 @@
...
+        subheading.This is home page subheading.This is home page subheading.This
+        is home page subheading.This is home page subheading.This is home page
+        subheading.This is home page subheading.This is home page subheading.This
+        is home page subheading.This is home page subheading.This is home page
+        subheading.This is home page subheading.This is home page subheading.This
+        is home page subheading.This is home page subheading.This is home page
+        subheading.This is home page subheading.
+      </p>
+    </body>
+</html>
```
Comment (avinash0070) [inline Vishnu/HTML/Assignment3/ass2_home.html]: add new lines at the end of the code

File: Vishnu/HTML/Assignment3/ass2_mean.html [hunk 18]
```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>MI-MEAN</title>
+        <style>
+            body {
+              background-color: #593703;
+              color: #6fa6a0;
+            }
```
Comment (avinash0070) [inline Vishnu/HTML/Assignment3/ass2_mean.html]: add new line

File: Vishnu/HTML/Assignment3/ass2_mean.html [hunk 19]
```diff
@@ -0,0 +1,42 @@
...
+            a:link {
+              color: #f2d4c2;
+            }
```
Comment (avinash0070) [inline Vishnu/HTML/Assignment3/ass2_mean.html]: add new line

File: Vishnu/HTML/Assignment3/ass2_mean.html [hunk 20]
```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
```
Comment (avinash0070) [inline Vishnu/HTML/Assignment3/ass2_mean.html]: need proper code indentation and remove all extra lines

File: Vishnu/HTML/Assignment3/ass2_mean.html [hunk 21]
```diff
@@ -0,0 +1,42 @@
...
+                <li>Hruday</li>
+                <li>Aniket</li>
+            </ul>
+            <ol>
+                <li>Bipin</li>
+                <li>Priyanka</li>
+                <li>Yashasvi</li>
+              </ol>
+    </body>
+</html>
```
Comment (avinash0070) [inline Vishnu/HTML/Assignment3/ass2_mean.html]: Add new line

File: Vishnu/HTML/Assignment4/Assign_1.html [hunk 22]
```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
```
Comment (avinash0070) [inline Vishnu/HTML/Assignment4/Assign_1.html]: need 4 spaces and proper code indentation

File: Vishnu/HTML/Assignment4/Assign_1.html [hunk 23]
```diff
@@ -0,0 +1,44 @@
...
+      </tr>
+      <tr>
+        <th>Females</th>
+        <td>1.7</td>
+        <td>0.007</td>
+        <td>43%</td>
+      </tr>
+    </table>
+</body>
+</html>
```
Comment (avinash0070) [inline Vishnu/HTML/Assignment4/Assign_1.html]: new line added

File: Vishnu/HTML/Assignment4/Assign_2.html [hunk 24]
```diff
@@ -0,0 +1,98 @@
...
+          <td></td>
+        </tr>
+        <tr>
+          <th>4:30</th>
+          <td></td>
+          <td></td>
+          <td></td>
+        </tr>
+    </table>
+</html>
```
Comment (avinash0070) [inline Vishnu/HTML/Assignment4/Assign_2.html]: add new line

File: Vishnu/HTML/Assignment4/Assign_2.html [hunk 25]
```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
```
Comment (avinash0070) [inline Vishnu/HTML/Assignment4/Assign_2.html]: need 4 spaces and proper code indentation

File: Vishnu/HTML/Assignment4/Assign_3.html [hunk 26]
```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
```
Comment (avinash0070) [inline Vishnu/HTML/Assignment4/Assign_3.html]: need 4 spaces and proper code indentation

File: Vishnu/HTML/Assignment4/Assign_3.html [hunk 27]
```diff
@@ -0,0 +1,76 @@
...
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Table_3</title>
+    <style>
+       table,
+      th,
+      td {
+        border: solid 1px black;
+      }
```
Comment (avinash0070) [inline Vishnu/HTML/Assignment4/Assign_3.html]: add new line in css code

File: Vishnu/HTML/Assignment4/Assign_3.html [hunk 28]
```diff
@@ -0,0 +1,76 @@
...
+        </tr>
+        <tr>
+          <th colspan="2">Schedule</th>
+          <th rowspan="2">Topic</th>
+        </tr>
+        <tr>
+          <th>Begin</th>
+          <th>End</th>
+        </tr>
+
```
Comment (avinash0070) [inline Vishnu/HTML/Assignment4/Assign_3.html]: remove extra line

File: Vishnu/HTML/Assignment4/Assign_3.html [hunk 29]
```diff
@@ -0,0 +1,76 @@
...
+        <tr>
+          <td class="yellow">11:00 a.m.</td>
+          <td class="green">2:00 p.m.</td>
+        </tr>
+        <tr>
+          <td class="green">2:00 p.m.</td>
+          <td class="purple">5:00 p.m.</td>
+          <td class="center">XSL Transformations</td>
+        </tr>
+
```
Comment (avinash0070) [inline Vishnu/HTML/Assignment4/Assign_3.html]: remove extra line

File: Vishnu_45/html/Assignment1/Assign2.html [hunk 30]
```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
```
Comment (avinash0070) [inline Vishnu_45/html/Assignment1/Assign2.html:1]: need proper code indentation

File: Vishnu_45/html/Assignment1/Assign3.html [hunk 31]
```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
```
Comment (avinash0070) [inline Vishnu_45/html/Assignment1/Assign3.html:1]: need proper code indentation

File: Vishnu_45/html/Assignment2/Assignment2.html [hunk 32]
```diff
@@ -0,0 +1,29 @@
...
+        <p>They called this search engine Back rub.</p>
+        <hr>
+        <h2>Initial public offering</h2>
+        <p>Google's initial public offering (IPO) took place five years later, on 19th August 2004.<br/>
+            At that time Larry Page, Sergey Brin, and Eric Schmidt agreed to work together at Google for 30 20<br/>
+            years, until the year 2024.
+        </p>
+        <h1>Thank You...</h1>
+    </body>
+</html>
```
Comment (avinash0070) [inline Vishnu_45/html/Assignment2/Assignment2.html]: add new line

File: Vishnu_45/html/Assignment3/ass1.html [hunk 33]
```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
```
Comment (avinash0070) [inline Vishnu_45/html/Assignment3/ass1.html:1]: need 4 spaces in code indentation
remove extra lines

File: Vishnu_45/html/Assignment5/style.css [hunk 34]
```diff
@@ -0,0 +1,38 @@
+body {
+  background: 1px solid black;
+  background-color: rgb(165, 106, 38);
+}
```
Comment (avinash0070) [inline Vishnu_45/html/Assignment5/style.css:4]: add new line 
change all with at the end of css property ends

File: Vishnu_45/html/Assignment5/index.html [hunk 35]
```diff
@@ -0,0 +1,35 @@
+<html>
```
Comment (avinash0070) [inline Vishnu_45/html/Assignment5/index.html:2]: need proper indentation

File: Vishnu_45/html/Assignment4/Assign_3.html [hunk 36]
```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
```
Comment (avinash0070) [inline Vishnu_45/html/Assignment4/Assign_3.html:1]: need proper indentation
add new line
change all with at the end of css property ends

File: Vishnu_45/html/Assignment4/Assign_2.html [hunk 37]
```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
```
Comment (avinash0070) [inline Vishnu_45/html/Assignment4/Assign_2.html:1]: need proper code indentation
remove all extra lines
add new line
change all with at the end of css property ends

File: Vishnu_45/html/Assignment4/Assign_1.html [hunk 38]
```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
```
Comment (avinash0070) [inline Vishnu_45/html/Assignment4/Assign_1.html:1]: need proper code indentation

File: Vishnu_45/html/Assignment3/ass2_mean.html [hunk 39]
```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
```
Comment (avinash0070) [inline Vishnu_45/html/Assignment3/ass2_mean.html:1]: need proper code indentation
add new line
change all with at the end of css property ends

File: Vishnu_45/html/Assignment3/ass2_home.html [hunk 40]
```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
```
Comment (avinash0070) [inline Vishnu_45/html/Assignment3/ass2_home.html:1]: need proper code indentation
remove all extra lines
add new line
change all with at the end of css property ends

File: Vishnu_45/html/Assignment3/ass2_contact.html [hunk 41]
```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
```
Comment (avinash0070) [inline Vishnu_45/html/Assignment3/ass2_contact.html:1]: need proper code indentation
remove all extra lines
add new line
change all with at the end of css property ends
remove  !DOCTYPE html

**Review by avinash0070 — CHANGES_REQUESTED** [review]

**Comment by avinash0070** [comment]
resolve all changes if they are done

**Comment by avinash0070** [comment]
resolve all the comments 
change title as well 
mention which file you added n your pull request

**Comment by avinash0070** [comment]
do changes as soon as possible

**Review by avinash0070 — APPROVED** [review]

**Comment by vishnukopnar** [comment]
successfully merged

**Pull request merged**